light-zk.js/tests/createOutUtxos.test.ts

```diff
@@ -682,7 +662,7 @@ describe("Test createOutUtxos Errors", () => {
     F = babyJub.F;
     k0 = new Account({ poseidon, seed: seed32 });
     k00 = new Account({ poseidon, seed: seed32 });
-    kBurner = Account.createBurner(poseidon, seed32, new BN("0"));
+    kBurner = Account.createBurner(poseidon, seed32, BN_0);
     splAmount = new BN(3);
```

Comment:
was there a particular reason for why we were instantiating the BN with a string instead of a number? (cc @ananas-block )

Reply:
there is no difference